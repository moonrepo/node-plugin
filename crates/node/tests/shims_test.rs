#[cfg(not(windows))]
use proto_pdk_test_utils::*;

#[cfg(not(windows))]
generate_shims_test!("node-test");
