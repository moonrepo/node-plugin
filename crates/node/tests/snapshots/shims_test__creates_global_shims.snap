---
source: crates/node/tests/shims_test.rs
expression: "std::fs::read_to_string(sandbox.path().join(\".proto/shims\").join(if cfg!(windows)\n                    {\n                    format!(\"{}.cmd\", \"node-test\")\n                } else { \"node-test\".to_string() })).unwrap()"
---
#!/usr/bin/env bash
set -e

if [ -n "$PROTO_DEBUG" ]; then
    set -x
    echo "Running with node-test.sh shim"
fi

exec proto run node-test --  "$@" 

